src/visions/application/summaries/series/path_summary.py

```diff
@@ -13,12 +13,18 @@ def path_summary(series: pd.Series) -> dict:
 
     """
     summary = {
-        "common_prefix": (os.path.commonprefix(list(series)) or "No common prefix"),
-        "stem_counts": series.map(lambda x: x.stem).value_counts().to_dict(),
-        "suffix_counts": (series.map(lambda x: x.suffix).value_counts().to_dict()),
-        "name_counts": series.map(lambda x: x.name).value_counts().to_dict(),
-        "parent_counts": (series.map(lambda x: x.parent).value_counts().to_dict()),
-        "anchor_counts": (series.map(lambda x: x.anchor).value_counts().to_dict()),
+        "common_prefix": os.path.commonprefix(list(series)) or "No common prefix",
+        "stem_counts": series.map(lambda x: x.stem).value_counts(),
+        "suffix_counts": series.map(lambda x: x.suffix).value_counts(),
+        "name_counts": series.map(lambda x: x.name).value_counts(),
+        "parent_counts": series.map(lambda x: x.parent).value_counts(),
+        "anchor_counts": series.map(lambda x: x.anchor).value_counts(),
```

Comment:
I'm not particularly familiar with the summary portions of the codebase but the changes from `value_counts().to_dict()` to `value_counts()` represents a breaking API change right? Effectively a change from dict to series on the return? Is that breaking anywhere else?